server/engine/src/main/java/org/eclipse/lsp/cobol/lsp/SourceUnitGraph.java

```diff
@@ -187,10 +187,6 @@ public boolean isCopybook(String uri) {
                 copyUri = copyUri.replace(" ", "%20");
                 return new URL(decodeUri).sameFile(new URL(copyUri));
               } catch (IOException e) {
-                if (ImplicitCodeUtils.isImplicit(copyUri)) {
```

Comment:
Where did the check for implicit URIs go? Shouldn't it be in there?

Reply:
It was just there for just logging purpose, which is redundant coz the other logger displays the same value.